db/schema.rb

```diff
@@ -96,6 +96,19 @@
     t.index ["user_id"], name: "index_profiles_on_user_id", unique: true
   end
 
+  create_table "ratings", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer "rate"
+    t.bigint "user_id", null: false
+    t.integer "reviewed_id", null: false
+    t.string "ratable_type"
+    t.bigint "ratable_id"
+    t.index ["ratable_type", "ratable_id"], name: "index_ratings_on_ratable_type_and_ratable_id"
+    t.index ["reviewed_id"], name: "index_ratings_on_reviewed_id"
```

Comment:
instead of ratable_type, ratable_id

(ratable_type, ratable_id, user_id, reviewed_id) unique: true